cmd/flexible-informer-gen/generators/informer.go

```diff
@@ -103,6 +103,8 @@ func (g *informerGenerator) GenerateType(c *generator.Context, t *types.Type, w
 		"v1ListOptions":                   c.Universe.Type(v1ListOptions),
 		"version":                         namer.IC(g.groupVersion.Version.String()),
 		"watchInterface":                  c.Universe.Type(watchInterface),
+		"isNotFound":                      c.Universe.Type(isNotFound),
+		"newFakeWatch":                    c.Universe.Type(newFakeWatch),
```

Comment:
'NewEmptyWatch' sounds like a better function to choose. However, reflector throws another error for it that causes the infinite wait.